Lesson1 (Sequence containers)/TaskFunctions.h

```diff
@@ -0,0 +1,14 @@
+#ifndef TASKFUNCTIONS_H
+#define TASKFUNCTIONS_H
+#include <iostream>
+#include <vector>
+#include <forward_list>
+
+std::vector<int> combineVectors(std::vector<int>::const_iterator begin1, std::vector<int>::const_iterator end1,
```

Comment:
**why these methods are in the second hometask? please create new branch from the master**

Reply:
where should they be?

Reply:
in another branch.
you should create a new branch (branched from master) for each hometask

Reply:
but i have created G3

Reply:
looks like you've branched from the last homework. so, changes from last homework are included in homework2 branch.
and when you're comaring it with master, the diff is first homework + second